doc/apimanager-reference.md

```diff
@@ -140,6 +140,8 @@ One APIManager custom resource per project is allowed.
 | HTTPSProxy | `httpsProxy` | string | No | N/A | Specifies a HTTP(S) Proxy to be used for connecting to HTTPS services. Authentication is not supported. Format is: `<scheme>://<host>:<port>` (see [docs](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#https_proxy-https_proxy)) |
 | NoProxy | `noProxy` | string | No | N/A | Specifies a comma-separated list of hostnames and domain names for which the requests should not be proxied. Setting to a single `*` character, which matches all hosts, effectively disables the proxy (see [docs](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#no_proxy-no_proxy)) |
 | ServiceCacheSize | `serviceCacheSize` | int | No | N/A | Specifies the number of services that APICast can store in the internal cache (see [docs](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#apicast_service_cache_size)) |
+| PriorityClassName         | `priorityClassName`         | string                                                                                                                                   | No           | N/A                                                                                                                                            | Specifies the Pod priority (see [docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#podspec-v1-core:~:text=higher%20the%20priority.-,priorityClassName,-string))  |
```

Comment:
This is what I see from your link:

![Screenshot 2023-06-06 at 15-39-20 Kubernetes API Reference Docs](https://github.com/3scale/3scale-operator/assets/881529/2c60f4a6-90a9-469b-84cc-4f00f4b7e190)

As description, I would say what its in the [podspec-v1-core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#podspec-v1-core), i.e.
```
If specified, indicates the pod's priority. "system-node-critical" and "system-cluster-critical" are two special keywords which indicate the highest priorities with the former being the highest priority. Any other name must be defined by creating a PriorityClass object with that name. If not specified, the pod priority will be default or zero if there is no default.
```

As link, I would include the k8s official doc about [pod priority](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/)

Reply:
when I tried this link -  https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#podspec-v1-core:~:text=higher%20the%20priority.-,priorityClassName,-string  -- It show less than second screen as you see (beginning of the doc),but after this it jump to following (or near following in [podspec-v1-core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#podspec-v1-core),  And I can see this image: 
![Screenshot from 2023-06-06 18-21-06](https://github.com/3scale/3scale-operator/assets/56625811/489e42b6-7ad6-4738-8600-e7c5f984c6f5)

Reply:
Done. Doc updated. Thank you for comment.